README.md

```diff
@@ -16,7 +16,7 @@ This application will use an OAuth Authorization Code workflow to log users in a
 
 2. Clone this repository.
 
-3. Configure your app in the FusionAuth admin panel (default [localhost:9011](localhost:9011)).
+3. Configure your app in the FusionAuth admin panel (default [localhost:9011](localhost:9011)). Copy the client Id, client secret, and application Id values from the FusionAuth admin panel to `client/src/config.js`.
 
 4. Make sure FusionAuth is running, then install dependencies and start the app. The React app should automatically open in your browser at [localhost:3000](http://localhost:3000).
```

Comment:
For Step 1 - make sure you have FusionAuth downloaded, should we recommend a particular option such as fast path?

Reply:
I fixed the link to be to the fast path install. Should I mention Kickstart?